source/Ship.cpp

```diff
@@ -922,6 +922,30 @@ void Ship::Save(DataWriter &out) const
 			for(const auto &it : baseAttributes.Attributes())
 				if(it.second)
 					out.Write(it.first, it.second);
+			// TODO: Finish output
+			for(const auto &it : baseAttributes.Factories())
```

Comment:
Why should we write-out the factories from the baseAttributes? (I would assume that we only need the factory data from the outfits.)